ts/boot/configureStoreAndPersistor.ts

```diff
@@ -438,13 +438,12 @@ const rootPersistConfig: PersistConfig = {
   migrate: createMigrate(migrations, { debug: isDevEnv }),
   // Entities and features implement a persisted reduce that avoids persisting messages.
   // Other entities section will be persisted
-  blacklist: ["entities", "features", "lollipop"],
+  blacklist: ["debug", "entities", "features", "lollipop"],
```

Comment:
Is this intended?

Reply:
Yes, I moved the debug persistence configuration to its own persistent, otherwise we would persist the debug data too.